avocado/plugins/xunit.py

```diff
@@ -35,7 +36,8 @@ def configure(self, parser):
 
         self.parser = parser
         run_subcommand_parser.output.add_argument(
-            '--xunit', type=str, dest='xunit_output', metavar='FILE',
```

Comment:
Sure, I'd keep one functional test for that. But IIRC there were several tests (maybe some were already removed, though)